app/src/main/java/edu/umd/cmsc436/mstestsuite/IntroActivity.java

```diff
@@ -35,9 +39,11 @@
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
```

Comment:
Just one last style change: Change the status bar color to white with the line `getWindow().setStatusBarColor(ContextCompat.getColor(this, android.R.color.white));` (It looks really slick)